helpers/twitter.ts

```diff
@@ -33,3 +34,16 @@ async function postTweet(status: string) {
     console.error("Error posting tweet", error);
   }
 }
+
+async function deleteTweet(id: string) {
+  try {
+    const { data } = await twitterClient.v2.deleteTweet(id);
+    if (data.deleted) {
```

Comment:
Pretty sure there's a bug here

https://github.com/BeanieMen/devpool-directory/actions/runs/7940849501/job/21682570537#step:7:12

Reply:
that was a redundant log removed in https://github.com/ubiquity/devpool-directory/pull/1046/commits/e87aecb79c0451efcc7e08b317da3712568ec396

Reply:
its showing undefined because i logged it after deleting key-value pair from twitterMap (ik i am dumb)